examples/hello/restful-hello-world.go

```diff
@@ -16,7 +16,10 @@ func main() {
 	ws := new(restful.WebService)
 	ws.Route(ws.GET("/hello").To(hello))
 	restful.Add(ws)
-	log.Fatal(http.ListenAndServe(":8080", nil))
+
```

Comment:
good point!

Reply:
thanks, for anyone without context, here is more reading: https://pkg.go.dev/net/http#Server.Shutdown

> When Shutdown is called, [Serve](https://pkg.go.dev/net/http#Serve), [ListenAndServe](https://pkg.go.dev/net/http#ListenAndServe), and [ListenAndServeTLS](https://pkg.go.dev/net/http#ListenAndServeTLS) immediately return [ErrServerClosed](https://pkg.go.dev/net/http#ErrServerClosed). Make sure the program doesn't exit and waits instead for Shutdown to return.